aten/src/ATen/native/Distributions.cpp

```diff
@@ -302,12 +302,27 @@ Tensor multinomial_cpu(const Tensor& self, int64_t n_sample, bool with_replaceme
   return result;
 }
 
+/* The largest consecutive integer representable in float32 (2^24) */
+#define FLOAT32_MAX_CONSECUTIVE_INT 16777216.0f
```

Comment:
This should be a `constexpr float` for type safety. In general I think we prefer `constexpr` to defining global variables as macros.

Reply:
Changed to constexpr int64_t FLOAT32_MAX_CONSECUTIVE_INT = 16777216